nixos/tests/postgresql.nix

```diff
@@ -1,26 +1,51 @@
-import ./make-test.nix ({ pkgs, ...} : {
+import ./make-test.nix ({ pkgs, ...} : 
+let
+  postgresql-versions = [
```

Comment:
Hi @spinus good to see this patch! Instead of listing all the versions manually couldn't you use the set:

```nix
pkgs.callPackages ../../../pkgs/servers/sql/postgresql { }
```

? This would have the advantage that new postgresql versions will be tested automatically.